app/views/users/webauthn_setup/new.html.slim

```diff
@@ -11,6 +11,8 @@ div
     = hidden_field_tag :user_email, current_user.email, id: 'user_email'
     = hidden_field_tag :user_challenge,
           '[' + user_session[:webauthn_challenge].split.join(',') + ']', id: 'user_challenge'
+    = hidden_field_tag :exclude_credentials, @exclude_credentials&.join(','),
```

Comment:
Since this is data being put in the page rather than a field being populated, would it make better sense to create a `data-` attribute and read that? This isn't a deal breaker. My personal preference would be to use `data-` attributes for reading and hidden fields for data being sent back to the server.